x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -194,6 +203,10 @@ export function isCurrentRelatedEventLoading(state: DataState) {
   return state.currentRelatedEvent.loading;
 }
 
+export function dataRefreshRequestsMade(state: DataState) {
```

Comment:
Would you add a doc comment for this please? Also, thoughts on the name `refreshCount` instead of `dataRefreshRequestsMade`?

Reply:
Yeah like that better, will change.